contents/docs/surveys/branching-logic.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Branching logic
+sidebar: Docs
+showTitle: true
+availability:
+  free: none
+  selfServe: full
+  enterprise: full
+---
+
+It is useful to be able to display survey questions conditionally. For example, you may want to ask certain questions based on the responses received. Branching logic allows you to do this.
+
+## Simple branching
+
+For every question type, you can now define the next step to navigate to after a question is completed. Click on __After this question, go to:__, then choose the next step you want to transition to. The options are:
+
+- Next question
+- Any specific question
+- Confirmation message (if present) or End
+
+After the given question is completed, the chosen next step will be followed.
+
+## Response-based branching
+
+For the __Single choice__ and __Rating__ question types, you can define the next step based on the response received. Click on __After this question, go to:__, and choose the option __Specific question based on answer__. Depending on your question type, this will present the following options.
+
+For the __Single choice__ question type, you can specify the next step based on the selected answer.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/surveys_branching_singlechoice_8053dd1700.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/surveys_branching_singlechoice_dark_0ec63f974a.png"
+    alt="Survey single choice branching" 
+    classes="rounded"
+/>
+
+For the __Rating__ question type, you can specify the next step based on the response bucket received. For example, for a rating question with a scale from 1 to 5, the responses will be bucketed into categories: _negative_, _neutral_, and _positive_. The branching will be executed based on which bucket the individual response falls into.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2024_07_03_at_14_09_29_210ff7ced4.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2024_07_03_at_14_10_37_7804a733b4.png"
+    alt="Survey rating branching" 
+    classes="rounded"
+/>
+
+
+__Note__: If you intend for a question to be the last one, but it is not the final question in the list, you need to explicitly set the branching to the __Confirmation message__ (if present) or __End__. By default, a question will branch to the next question unless specified otherwise.
```

Comment:
```suggestion
By default, a question proceed to the next question in the survey unless specified otherwise. If you want a question to be the last one, but it is not the final question in the list, you need to explicitly set the value for **After this question go to ** to __Confirmation message__ (if present) or __End__..
```